tests/external_executor/external_executor.py

```diff
@@ -101,7 +101,10 @@ def register_new_executor(node, network, message=None):
     return executor_credentials
 
 
-@reqs.description("Register an external executor")
+@reqs.description(
+    "Register an external executor (Disabled on SNP due to UNKNOWN RPC failures)"
```

Comment:
Could you create a Github issue for this and link it here? It becomes hard to keep track of special cases like this one otherwise.

Reply:
Done #4505